tests/test-posttype.php

```diff
@@ -22,7 +22,7 @@ class PostTypeTest extends \WP_UnitTestCase {
 	/**
 	 * @group posttypes
 	 */
-	function onNotSuccessfulTest( Throwable $t ) {
+	function onNotSuccessfulTest( Throwable $t ): void {
```

Comment:
@arzola This was a change in PHPUnit 8 or 9. All the other methods that now require :void had WordPress replacements using snake case (like setUp -> set_up), but not this one.